test/pages/cart.test.js

```diff
@@ -29,11 +32,24 @@ afterAll(() => {
 
 test('dispatch updateQuantity action on changing line item quantity', () => {
   // Arrange
-
   const expectedFunction = updateLineItemQuantity().toString()
   const updateQuantitySpy = jest.spyOn(CartPage.prototype, 'updateQuantity')
   const dispatch = jest.fn().mockImplementation((updateSpy) => 'first call')
 
+  const head = () => {
+    return (
+      <head>
+        <meta name='viewport' content='width=device-width, initial-scale=1, maximum-scale=1.0' />
+        <link rel='icon' type='image/png' sizes='32x32' href='../../static/favicon.png' key='favicon' />
+        <script src='https://js.stripe.com/v3/' key='stripe' />
+      </head>
```

Comment:
Just an observation - I think the head in specs doesn't need to mirror what we have in production exactly, we could keep it a bit simpler, e.g. leave out the stripe script or maybe even use the default empty head.

Reply:
👍